crypto/rand/rand_lib.c

```diff
@@ -333,6 +333,10 @@ const RAND_METHOD *RAND_get_rand_method(void)
 int RAND_priv_bytes_ex(OSSL_LIB_CTX *ctx, unsigned char *buf, size_t num,
                        unsigned int strength)
 {
+#ifdef __wasi__
```

Comment:
This would be better done by something in `providers/implementations/rands`.  Probably in the `seeding` subdirectory.  If what's there currently is good enough, remove these.

That is, instead of completely replacing OpenSSL RNG infrastructure, use it.

Reply:
Actually, I don't think we need it right now, reverted the change.